testsuite/geb-testsuite/src/test/java/GebConfig.groovy

```diff
@@ -0,0 +1,133 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/*
+ This is the Geb configuration file.
+
+ See: http://www.gebish.org/manual/current/configuration.html
+*/
+
+import java.util.logging.Level
+
+import geb.driver.SauceLabsDriverFactory
+import org.openqa.selenium.Dimension
+import org.openqa.selenium.Platform
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.chrome.ChromeDriver
+import org.openqa.selenium.firefox.FirefoxDriver
+import org.openqa.selenium.ie.InternetExplorerDriver
+import org.openqa.selenium.phantomjs.PhantomJSDriver
+import org.openqa.selenium.phantomjs.PhantomJSDriverService
+import org.openqa.selenium.remote.DesiredCapabilities
+import org.openqa.selenium.remote.RemoteWebDriver
+import org.openqa.selenium.safari.SafariDriver
+
+reportsDir = "target/test-reports/geb"
```

Comment:
This file allows for configuring the WebDriver and a couple of key framework parameters. Screenshots, etc captured during testing end up nested under it-data in this ^^^ folder